setup.sh

```diff
@@ -12,6 +12,6 @@ else
     pip install --upgrade --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu118
 fi
 # Build tools
-pip install black ruff pytest build
+pip install ruff pytest build
```

Comment:
This is a legacy file I should have removed a long time ago since it is obsoleted by optional targets in pyproject.toml.

Anyway, you also need to edit the `python-quality` CI workflows under `.github`.

Reply:
>Anyway, you also need to edit the python-quality CI workflows under .github.

Do we want similar verbosity flags on this one as well?
I see that currently it simply calls `check` :
`
# python-quality.yml
- run: ruff check bench examples optimum test
`

Reply:
Yes that would be useful thank you.

Reply:
> > Anyway, you also need to edit the python-quality CI workflows under .github.
> 
> Do we want similar verbosity flags on this one as well? I see that currently it simply calls `check` : `- run: ruff check bench examples optimum test`

On second thought, since this is an integration check, I think it should be fine without a listing flag...

Reply:
Alrighty then, done.

Reply:
The reason it might still be relevant is because ppl submitting pull-request might not be familiar with the concept of linting/formatting, but would understand immediately what this is about when looking at the failing CI logs for their pull-request.

Reply:
I see... yes, agreed.